static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,211 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MissingMember, Organization} from 'sentry/types';
+import withOrganization from 'sentry/utils/withOrganization';
+
+type Props = {
+  missingMembers: {integration: string; users: MissingMember[]};
+  onSendInvite: (email: string) => void;
+  organization: Organization;
+};
+
+export function InviteBanner({missingMembers, onSendInvite, organization}: Props) {
+  // NOTE: this is currently used for Github only
+  // TODO(cathy): include snoozing, docs link
+
+  if (
+    !organization.access.includes('org:write') ||
+    !missingMembers?.users ||
+    missingMembers?.users.length === 0
+  ) {
+    return null;
+  }
+
+  const cards = missingMembers?.users.slice(0, 5).map(member => (
+    <MemberCard key={member.userId} data-test-id={`member-card-${member.userId}`}>
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <IconGithub size="sm" />
+          {/* TODO: create mapping from integration to lambda external link function */}
+          <StyledExternalLink href={`http://github.com/${member.userId}`}>
+            {tct('@[userId]', {userId: member.userId})}
+          </StyledExternalLink>
+        </MemberCardContentRow>
+        <MemberCardContentRow>
+          <IconCommit size="xs" />
+          {tct('[commitCount] Recent Commits', {commitCount: member.commitCount})}
+        </MemberCardContentRow>
+        <Subtitle>{member.email}</Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        onClick={() => onSendInvite(member.email)}
```

Comment:
What happens if I double-click this button quickly? Is there a loading state that prevents duplicated requests?

Reply:
no but i will add one